StorySquad/SubmissionsAPI.swift

```diff
@@ -0,0 +1,429 @@
+//
+//  SubmissionsAPI.swift
+//  StorySquad
+//
+//  Created by Bohdan Tkachenko on 6/7/21.
+//  Copyright © 2021 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+// Put in global file
+enum HeaderNames: String {
+    case authorization = "Authorization"
+    case contentType = "Content-Type"
+}
+
+// Put in global file
+enum NetworkingError: Error {
+    case noBearer
+    case serverError(Error)
+    case badDecode
+    case unauthorized
+    case noData
+    case unexpectedStatusCode(Int)
+    case badEncoding
+    case noRepresentation
+    case formattedJSONIncorrectly
+    case missingRequiredElement
+}
+
+class SubmissionsAPI {
+    
+    private let baseURL = URL(string: "https://storysquad-api.herokuapp.com/")!
+    
+    var parentUser: Parent?
+    var children: [Child]?
+    var childUser: Child?
+    
+    //Include parent and child TOKENS to model
+    var parentBearer: Bearer?
+    var childBearer: Bearer?
+    
```

Comment:
Instead of passing in the submission, which you don't have yet, you'll need to access the current user's id and also fetch the storyId using the current user's cohortId.